templates/token-staking-dapp-template/README.md

```diff
@@ -0,0 +1,52 @@
+# Create Aptos Dapp Stake Pool Template
```

Comment:
should we unify naming? choose one from stake pool template and token staking dapp template?